wazo_call_logd/plugins/cdr/schemas.py

```diff
@@ -70,6 +79,7 @@ class CDRSchema(Schema):
     answer = fields.DateTime(attribute='date_answer')
     duration = fields.TimeDelta(default=None, attribute='marshmallow_duration')
     call_direction = fields.String(attribute='direction')
+    destination_details = fields.Nested(DestinationDetailsSchema, many=False)
```

Comment:
Not necessary to override default value
```suggestion
    destination_details = fields.Nested(DestinationDetailsSchema)
```